cmd/gh2gcs/cmd/root.go

```diff
@@ -197,9 +206,12 @@ func run(opts *options) error {
 		if err := gh2gcs.DownloadReleases(&rc, gh, opts.outputDir); err != nil {
 			return errors.Wrap(err, "downloading release assets")
 		}
+		logrus.Infof("Files downloaded to %s folder", opts.outputDir)
```

Comment:
One of the nittiest nits and non blocking: `folder` seems to be a term from the windows world. I would prefer if we stick to `directory` instead. 😋

Reply:
this shows my age lol from the 3.11 for workgroups